sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs

```diff
@@ -33,6 +33,16 @@ public class HealthcareOperation : Operation<RecognizeHealthcareEntitiesResultCo
         /// </summary>
         public override string Id { get; }
 
+        /// <summary>
+        /// Properties for healthcare operation
+        /// </summary>
+        public HealthcareOperationProperties Properties { get; set; }
```

Comment:
Added this variable to have the operation properties stored in a separate variable. KV is using a similar pattern. Currently, it is not being set but will add an issue to fix it.